Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/FlyoutSamplePage.xaml

```diff
@@ -118,51 +118,101 @@ turpis semper nisi maecenas, nascetur dictumst sed arcu aenean varius dis leo ha
 						</smtx:XamlDisplay>
 
 						<!-- Material MenuFlyout + MenuFlyoutSubItem -->
-						<smtx:XamlDisplay UniqueKey="FlyoutSamplePage_Material_Flyout"
-										  smtx:XamlDisplayExtensions.Header="Nested MenuFlyout">
+						<win:Grid>
+							<smtx:XamlDisplay UniqueKey="FlyoutSamplePage_Material_Flyout_win"
```

Comment:
We should probably toggle this via x:load=false + visibility (either via Binding or codebehind).
It would be easier to control complex condition, & without having to duplicate code.